nl_app.py

```diff
@@ -17,7 +17,7 @@
 
 import logging
 
-from nl_server.__init__ import create_app
```

Comment:
Do we want an empty `__init__.py` per python best practice or something?

Reply:
oh, did not meant to delete it.